docs/learn/glossary.md

```diff
@@ -110,7 +110,7 @@ Groups of tokens. Supply pools represent the total supply of tokens in a market.
 
 ## Proof of Stake
 
-Proof of Stake. A style of blockchain where validators are chosen to propose blocks according to the number of coins they hold.
+A blockchain consensus mechanism where validators are chosen to confirm blocks at a rate proportional to the number of tokens they hold.
```

Comment:
I noticed a contradiction in the original definition which should be rewritten. Validators never "hold" a delegator's coins in their stake. 

In Tendermint PoS, proposers are chosen by their stake. Then all other validators are allowed to confirm by signing, so it may be more appropriate to mention the proposer. 

Something along the lines of:
"A blockchain consensus mechanism where validators are chosen to propose blocks at a rate proportional to their [stake](#staking).  The [Tendermint Consensus](#tendermint-consensus) is the specific  Proof of Stake model used by Terra."